### PR TITLE
feat: delegate modules

### PR DESCRIPTION
**Pull request by ScriptedAlchemy**

### CODE REVIEW

**Comment by mdstaff** [comment]
👋 Hi there, I'm curious if you have any thoughts on when this would be ready to merge. 
Thanks much!

**Comment by zhangHongEn** [comment]
I just happened to be doing similar things recently, so I just copied the intent of this pr delegate-modules @ScriptedAlchemy 
[delegate-modules](https://github.com/zhangHongEn/universal-module-federation-plugin/tree/main/packages/universal-module-federation-plugin#delegate-modules)

File: delegate-modules/app1/webpack.config.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  // https://codesandbox.io/docs/environment
+  if (!CODESANDBOX_SSE) {
+    return `//localhost:${port}/remoteEntry.js`;
+  }
+
+  const parts = HOSTNAME.split('-');
+  const codesandboxId = parts[parts.length - 1];
+
+  return `//${codesandboxId}-${port}.sse.codesandbox.io/remoteEntry.js`;
+}
```
Comment (brunos3d) [inline delegate-modules/app1/webpack.config.js:69]: This function is not being used anywhere

**Review by brunos3d — DISMISSED** [review]
Just leave a comment, LGTM :rocket:

**Pull request merged**